info.limpet.rcp/src/info/limpet/rcp/data_provider/data/CollectionWrapper.java

```diff
@@ -18,42 +18,6 @@ public CollectionWrapper(final LimpetWrapper parent,
 		_collection = collection;
 	}
 
-	
```

Comment:
I needed to insert hashCode() and equals() in order for the JFace Treeview to remember which folders are open/collapsed when we do a refresh.

Did you intend to remove these methods, or is it because your branch was taken before they were added?